nucypher/utilities/prometheus/metrics.py

```diff
@@ -1,45 +1,38 @@
-
-from nucypher.exceptions import DevelopmentInstallationRequired
-
-try:
-    from prometheus_client.core import Timestamp
-    from prometheus_client.registry import REGISTRY, CollectorRegistry
-    from prometheus_client.utils import floatToGoString
-except ImportError:
-    raise DevelopmentInstallationRequired(importable_name='prometheus_client')
-
 import json
 from typing import List
 
+from prometheus_client import GC_COLLECTOR, PLATFORM_COLLECTOR, PROCESS_COLLECTOR
+from prometheus_client.core import Timestamp
+from prometheus_client.registry import REGISTRY, CollectorRegistry
+from prometheus_client.twisted import MetricsResource
+from prometheus_client.utils import floatToGoString
 from twisted.internet import reactor, task
 from twisted.web.resource import Resource
+from twisted.web.server import Site
 
 from nucypher.characters import lawful
 from nucypher.utilities.prometheus.collector import (
     BlockchainMetricsCollector,
     MetricsCollector,
-    OperatorMetricsCollector,
     StakingProviderMetricsCollector,
     UrsulaInfoMetricsCollector,
 )
 
 
 class PrometheusMetricsConfig:
     """Prometheus configuration."""
-    def __init__(self,
-                 port: int,
-                 metrics_prefix: str,
-                 listen_address: str = '',  # default to localhost ip
-                 collection_interval: int = 90,  # every 1.5 minutes
-                 start_now: bool = False):
 
+    def __init__(
+        self,
+        port: int,
+        listen_address: str = "",  # default to localhost ip
+        collection_interval: int = 90,  # every 1.5 minutes
```

Comment:
If this is not going to be a CLI option (previously `--metrics-interval`), perhaps it can be controlled by an environment variable whose default value is 90 if not specified.

The concern here is that prometheus usage can vary across nodes, and it is unclear whether there is one "correct" interval that applies to all usages. Obviously collecting data more often increases the number of RPC requests, and running less often decreases the number of requests; it's just a question of optionality for node oeprators.

Reply:
Fixed via #3388